stackslib/src/net/download/nakamoto/download_state_machine.rs

```diff
@@ -1760,13 +1680,10 @@ impl NakamotoDownloadStateMachine {
                     },
                 );
 
-                // keep borrow-checker happy by instantiang this ref again, now that `network` is
-                // no longer mutably borrowed.
-                let Some(invs) = network.inv_state_nakamoto.as_ref() else {
-                    // nothing to do
-                    debug!("No network inventories");
-                    return HashMap::new();
-                };
+                if throttled {
```

Comment:
as self.download_unconfirmed_tenures never deems to return throttled = true...can prob remove this.

Reply:
Yup